en/flight_controller/beaglebone_blue.md

```diff
@@ -57,7 +57,8 @@ connmanctl>scan wifi
 connmanctl>services
 #(at this point you should see your network SSID appear.)
 connmanctl>agent on
-connmanctl>connect <SSID>
+connmanctl>connect wifixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
I am not sure about this. Unless you literally enter `wifixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`, then I would leave as it was, and after this maybe add a note:

> Note, the format of the `<SSID>` is normally the text 'wifi' followed by a string of other characters. After entering the command you will be prompted to enter the wifi password.